### DISCUSSION TITLE
Collection system property

### DISCUSSION THREAD

**Discussion by mrbu-solutions** [pocketbase/pocketbase#2980] [Q&A] [answered]
Hi Pocketbase team,

What is collection's _system_ field for?

It says in the documentation:

![image](https://github.com/pocketbase/pocketbase/assets/53817867/e8c0447d-67de-42cb-bd4c-240437c04c43)

But when I try to update it via Web API, I've got a response error:

![image](https://github.com/pocketbase/pocketbase/assets/53817867/9b500e8b-e8fc-4277-a0cc-f64cb2fc661b)

**Note:** I'm logged in as an admin

Thank you very much.

**Comment by ganigeorgiev** [comment]
You can't change the collection `system` flag with the web APIs once it is set (see also the reply in your other discussion in https://github.com/pocketbase/pocketbase/discussions/2840).

_Note that the behavior of the collection and schema `system` field may change in the future._

**Reply by mrbu-solutions** [reply]
When you say "may change", does that mean that we will be able to update it in the future?

**Reply by ganigeorgiev** [reply, marked as answer]
There are plans eventually introducing it in the Admin UI and allow changing the field state with the web APIs so that we can have "lock/unlock" functionality, but for now remains a low priority.

**Reply by mrbu-solutions** [reply]
Thank you for the info. Appreciate it.
